pkg/pfcpsim/pfcpsim-client/pfcpsim-client.go

```diff
@@ -0,0 +1,168 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 Open Networking Foundation
+
+package pfcpsim_client
+
+import (
+	"github.com/c-robinson/iplib"
+	"github.com/omec-project/pfcpsim/pkg/pfcpsim"
+	log "github.com/sirupsen/logrus"
+	"github.com/wmnsk/go-pfcp/ie"
+	"net"
+	"time"
+)
+
+const (
+	ActionForward uint8 = 0x2
+	ActionDrop    uint8 = 0x1
+	ActionBuffer  uint8 = 0x4
+	ActionNotify  uint8 = 0x8
+)
+
+type MockSMF struct {
```

Comment:
This should be re-named. I propose `PFCPSim` (we use uppercase for acronyms in Go) or `PFCPSimClient`.

Reply:
Done